src/main/java/core/basesyntax/servise/strategy/OperationStrategies.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.servise.strategy;
+
+import core.basesyntax.servise.FruitTransaction;
+
+public interface OperationStrategies {
```

Comment:
```suggestion
public interface OperationStrategy {
```
it will have a lot of implementations, but still should be in singular